redisinsight/ui/src/components/item-list/ItemList.tsx

```diff
@@ -75,6 +75,12 @@ function ItemList<T extends { id: string; visible?: boolean }>({
       return
     }
 
+    if (!instances.length) {
+      // TODO: add proper message about no instances
```

Comment:
Is this a `TODO` we could resolve in this card or does it need to be in a separate card?

Reply:
No, it will be covered in another story - welcome screen for database list page